test/services/helpers/services/schoolGroups.js

```diff
@@ -1,24 +1,24 @@
-const { schoolGroupModel } = require('../../../../src/services/school/model');
-
-let createdSchoolGroups = [];
-
-const create = async (data) => {
-	const schoolGroup = await schoolGroupModel.create(data);
-	createdSchoolGroups.push(schoolGroup._id);
-	return schoolGroup;
-};
-
-const cleanup = async () => {
-	if (createdSchoolGroups.length === 0) {
-		return Promise.resolve();
-	}
-	const ids = createdSchoolGroups;
-	createdSchoolGroups = [];
-	return schoolGroupModel.deleteMany({ _id: { $in: ids } });
-};
-
-module.exports = {
-	create,
-	cleanup,
-	info: () => createdSchoolGroups,
-};
+const { schoolGroupModel } = require('../../../../src/services/school/model');
+
+let createdSchoolGroups = [];
+
+const create = async (data) => {
+	const schoolGroup = await schoolGroupModel.create(data);
+	createdSchoolGroups.push(schoolGroup._id);
+	return schoolGroup;
+};
+
+const cleanup = async () => {
+	if (createdSchoolGroups.length === 0) {
+		return Promise.resolve();
+	}
+	const ids = createdSchoolGroups;
+	createdSchoolGroups = [];
+	return schoolGroupModel.deleteMany({ _id: { $in: ids } }).lean().exec();
+};
+
+module.exports = {
+	create,
+	cleanup,
+	info: () => createdSchoolGroups,
+};

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5258109)